tasks/tests/test_view_single_task.py

```diff
@@ -0,0 +1,34 @@
+from tasks.models import Status, Task
+from tasks.forms import TaskForm
+import pytest
+
+
+@pytest.mark.django_db
+class TestViewSingleTask:
+
+    def test_no_authenticated_user_asks_task(self, client, task_1):
+        response = client.get(f'/tasks/{task_1.id}')
+        assert response.status_code == 302
+        assert response.url == '/'
+
+    def test_with_employee_logged_in(self, client, task_1, employee_1):
+        client.login(username=employee_1.user.username,
+                     password='password')
```

Comment:
I don't think you need to break it into a few lines
unless its many variables with a long line for each variable:
```suggestion
        client.login(username=employee_1.user.username, password='password')
```